frontend/lib/src/components/shared/BaseButton/styled-components.ts

```diff
@@ -224,16 +227,29 @@ export const StyledSecondaryFormSubmitButton = styled(
 export const StyledIconButton = styled(
   StyledBaseButton
 )<RequiredBaseButtonProps>(({ size, theme }) => {
-  const iconPadding: Record<BaseButtonSize, string> = {
-    [BaseButtonSize.XSMALL]: theme.spacing.threeXS,
-    [BaseButtonSize.SMALL]: theme.spacing.twoXS,
-    [BaseButtonSize.MEDIUM]: theme.spacing.md,
-    [BaseButtonSize.LARGE]: theme.spacing.lg,
+  // const iconPadding: Record<BaseButtonSize, string> = {
+  //   [BaseButtonSize.XSMALL]: theme.spacing.threeXS,
+  //   [BaseButtonSize.SMALL]: theme.spacing.twoXS,
+  //   [BaseButtonSize.MEDIUM]: theme.spacing.md,
+  //   [BaseButtonSize.LARGE]: theme.spacing.lg,
+  // }
+  console.log("size", size)
+  let minWidth = "12rem"
+  if (size === BaseButtonSize.SMALL) {
+    minWidth = "8rem"
+  } else if (size === BaseButtonSize.LARGE) {
+    minWidth = "20rem"
   }
+
   return {
     backgroundColor: theme.colors.transparent,
-    border: `${theme.sizes.borderWidth} solid ${theme.colors.transparent}`,
-    padding: iconPadding[size],
+    border: `${theme.sizes.borderWidth} solid ${theme.colors.borderColor}`,
+    // padding: iconPadding[size],
+    flex: "1 1 0",
+    padding: 0,
+    maxWidth: minWidth, //theme.sizes.contentMaxWidth,
+    minWidth: minWidth,
+    // width: "max-content",
```

Comment:
1. nit: should we clean up the unused code? 
2. This should not change st.feedback's layout, right? (given `st_feedback.py` passes)